cmd/access/staked_access_node_builder.go

```diff
@@ -32,3 +48,48 @@ func (sanb *StakedAccessNodeBuilder) Initialize() cmd.NodeBuilder {
 
 	return sanb
 }
+
+// enqueueUnstakedNetworkInit enqueues the unstaked network component initialized for the staked node
+func (sanb *StakedAccessNodeBuilder) enqueueUnstakedNetworkInit() {
+
+	sanb.Component("unstaked network", func(node cmd.NodeBuilder) (module.ReadyDoneAware, error) {
+
+		// NodeID for the staked node on the unstaked network
+		unstakedNodeID := sanb.NodeID() // currently set the same as the staked NodeID
+
+		// Networking key
+		unstakedNetworkKey := sanb.NetworkKey()
+
+		// Network Metrics
+		// for now we use the empty metrics NoopCollector till we have defined the new unstaked network metrics
+		unstakedNetworkMetrics := metrics.NewNoopCollector()
+
+		// intialize the LibP2P factory with an empty metrics NoopCollector for now till we have defined the new unstaked
+		// network metrics
+		libP2PFactory, err := sanb.FlowAccessNodeBuilder.initLibP2PFactory(unstakedNodeID, unstakedNetworkMetrics, unstakedNetworkKey)
+		sanb.MustNot(err)
+
+		// use the default validators for the staked access node unstaked networks
+		msgValidators := p2p.DefaultValidators(sanb.Logger(), unstakedNodeID)
+
+		// don't need any peer updates since this will be taken care by the DHT discovery mechanism
+		peerUpdateInterval := time.Hour
+
+		middleware := sanb.initMiddleware(unstakedNodeID, unstakedNetworkMetrics, libP2PFactory, peerUpdateInterval, msgValidators...)
+
+		// empty list of unstaked network participants since they will be discovered dynamically and are not known upfront
```

Comment:
Nit: technically, we would initialize with those other staked ANs that participate in the public network. That's probably better left to a follow-up PR, but a TODO comment here would be nice.

Reply:
@vishalchangrani The plan is to eventually have more than one staked AN that participates in the public network, right?

Reply:
eventually, yes. But initially I would like to just keep it to one staked AN which could support upto 10 to 15 unstaked AN.
Adding more than one ANs to the unstaked network would mean that all those AN also see unnecessary network traffic.